tests/unit/jira/test_service.py

```diff
@@ -180,7 +179,6 @@ def test_get_issue_reraises_other_erroring_status_codes(
             jira_service.get_issue(context=action_context, issue_key="JBI-234")
 
     [record] = capturelogs.records
-    assert record.levelno == logging.DEBUG
```

Comment:
Shouldn't we switch these to `logging.INFO` rather than deleting them?

Reply:
Hmm, why not, although this is not really something we assert usually